### ISSUE TITLE
Sonarr does not push notices when files are updated

### ISSUE THREAD

**Issue by salomj** [Sonarr/Sonarr#4630]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When a video file is updated, for example, subtitles are added to the video, Sonarr does not push updates to downstream services on rescan. Also, if I remove the series from Sonarr and then reimport the series, it will not update downstream services. 

### Expected Behavior

Sonarr should detect file updates and tell downstream services like Plex and Bazarr that the file has been changed. This could be done by checking the file date and the file size for changes.

### Steps To Reproduce

1. Import a series
2. Replace the video file with an updated video file. 
3. Notice that Sonarr does not pick up any changes.

There are several reasons why I'd want to update the files.
-  Pull unsupported subtitles out of video files, so that plex can direct play
- Merge supported subtitles back into the video files, so the folder structure is less messy. (I also very much prefer them as a single file as it makes it much easier to move the files around)
- Convert the video format to one that plex can direct play
- Convert the audio format to one that plex can direct play

### Environment

```markdown
- OS: Ubuntu 20.04
- Sonarr: 3.0.6.1265
- Docker Install: 3.0.6.1265-ls114 by linuxserver.io
- Using Reverse Proxy: No
- Browser: Chrome 92.0.4515.131
```

### What branch are you running?

Main

### Anything else?

See: https://github.com/morpheus65535/bazarr/issues/1502

**Comment by markus101** [comment]
Dupe of https://github.com/Sonarr/Sonarr/issues/3366
